daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DataProcessor.scala

```diff
@@ -692,40 +699,35 @@ class DataProcessor private (
 class ParseResult(dp: DataProcessor, override val resultState: PState)
   extends DFDL.ParseResult
   with WithDiagnosticsImpl
-  with ErrorHandler {
```

Comment:
This moved down into the default validator

Reply:
Or any validator right? A non-default one also has to provide an error handler, yes?

Reply:
Depends on the underlying validation, this sax handler is not required.  For example the Schematron validator doesnt use callbacks it just returns a list of results you sort through 

https://github.com/ctc-oss/daffodil-schematron-validator/blob/master/src/main/scala/com/ctc/daffodil/ext/SchematronValidator.scala#L19-L22